File: UtilitiesCS/Interfaces/IToDoProjectInfoEntry.cs

using System;

namespace UtilitiesCS
{
    public interface IToDoProjectInfoEntry
    {
        string ProgramName { get; set; }
        string ProjectID { get; set; }
        string ProjectName { get; set; }
        int CompareTo(IToDoProjectInfoEntry other);
        bool Equals(object obj);
        bool Equals(IToDoProjectInfoEntry other);
        string ToCSV();
        bool IsAnyNull();
        void SetIdUpdateAction(Action<string, string> action);
    }
}

File: UtilitiesVB/Solution Interfaces/IToDoProjectInfoEntry.vb
Public Interface IToDoProjectInfoEntry
    Property ProgramName As String
    Property ProjectID As String
    Property ProjectName As String
    Function CompareTo(other As IToDoProjectInfoEntry) As Integer
    Function Equals(obj As Object) As Boolean
    Function Equals(other As IToDoProjectInfoEntry) As Boolean
    Function ToCSV() As String
End Interface
